src/opcodes/opcodes.cairo

```diff
@@ -174,6 +175,17 @@ pub mod Opcode {
         engine.dstack.push_int(a + b);
     }
 
+    fn opcode_not(ref engine: Engine) {
+        let m = engine.dstack.pop_int();
+        if m == 0 {
+            engine.dstack.push_int(1);
+        } else if m == 1 {
+            engine.dstack.push_int(0);
+        } else {
+            engine.dstack.push_int(0);
+        }
```

Comment:
Unnecessary else if m == 1 you can just condense it into the else

Reply:
yaah, i will fixed and push